train_protection.mss

```diff
@@ -47,8 +49,7 @@
   [zoom>=13]["railway"="light_rail"],
   [zoom>=11]["railway"="tram"]["service"=null],
   [zoom>=13]["railway"="tram"] {
-    ["pzb"="no"]["lzb"="no"]["etcs"="no"],
-    ["atb"="no"]["etcs"="no"] {
+    ["pzb"="no"]["lzb"="no"]["atb"="no"]["kvb"="no"]["tvm"="no"]["scmt"="no"]["etcs"="no"] {
```

Comment:
This change is wrong for the same reason as stated above. You need to add a case where tvm == kvb == etcs == no to match French lines without any train protection.